saleor/graphql/app/types.py

```diff
@@ -264,7 +300,7 @@ class App(ModelObjectType):
     )
     extensions = NonNullList(
         AppExtension,
-        description="App's dashboard extensions." + ADDED_IN_31 + PREVIEW_FEATURE,
+        description=f"App's dashboard extensions.{ADDED_IN_31}{PREVIEW_FEATURE}",
```

Comment:
Why are we changing this? The previous format was the one commonly used.

Reply:
done, reverted.